src/Components/PasswordRules.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace ARKEcosystem\Fortify\Components;
+
+use Illuminate\View\Component;
+
+class PasswordRules extends Component
```

Comment:
Does this have to be a class component if we are not doing anything with the `passwordRules` array inside of this class?